src/components/Button/Button.jsx

```diff
@@ -17,7 +18,26 @@ export function Button({
   onClick,
   className,
   type,
+  to = false,
 } = {}) {
+  if (to)
+    return (
+      <Link
+        className={classNames(
+          styles.button,
+          variantToClass[variant],
+          className,
+          {
+            [styles.fullw]: fullwidth,
+          },
+        )}
+        to={to}
+        onClick={onClick}
```

Comment:
do we need `onClick` for `Link`?

Reply:
Yes, it's fully semantic, it's clickable element and I needed it for one case

Reply:
Ok, now I got it. In that case, maybe it would be more suitable not to treat that button as anchor if it's executing some logic -> call both `deleteIngredient` and `navigate` in `onClick`?